makeinvoice.go

```diff
@@ -33,7 +33,9 @@ type Params struct {
 	Description     string
 	DescriptionHash []byte
 
-	Label string // only used for c-lightning
+	Label    string // only used for c-lightning
+	Username string // only used for strike
+	Currency string // only used for strike
```

Comment:
can you put these two Strike params on `StrikeParams`?

Reply:
I didn't do that because the provider (satdress or some app) can request invoices on behalf of others right? so isn't it better to keep it as input param? so key and username/currency should be in different places. at least that's what I thought.

Reply:
well, some params are specified only once at configuration time, others are specified for each invoice. I see how you could have these in either place, but I think it makes more sense to define which user is going to receive the payments at configuration time, since that's how all the other backends work (implicitly).

Reply:
understood. will update.